classes/Divisions.php

```diff
@@ -257,10 +257,16 @@ public function getMemberDivisionDetails($strong_only = false) {
             GROUP BY policy_id, policy_vote, vote",
             $args
         );
+        
+        $allowed_policies = (new \MySociety\TheyWorkForYou\Policies())->getPolicyIDs();
 
         foreach ($q as $row) {
           $policy_id = $row['policy_id'];
-
+          // deleted policies may still be in the database - ignore them
```

Comment:
This feels like it's a bit more than the commit message implies - mention this in the commit?